apis/python/src/tiledbsoma/io/_registration/ambient_label_mappings.py

```diff
@@ -233,21 +233,23 @@ def from_isolated_soma_experiment(
                     obs_map.update(dict(zip(obs_ids, soma_joinids)))
 
                 for measurement_name in exp.ms:
-                    exp.ms[measurement_name].var
+                    expvar = exp.ms[measurement_name].var
                     if "var" not in exp.ms[measurement_name]:
                         continue
+                    if var_field_name not in expvar.schema.names:
```

Comment:
It would be possible to not even consult all measurments -- only the one of interest higher up in the call stack.

However, this isn't neat -- since there is not _one_ of interest. At multiple spots in today's ingestor, we take the current measurement name _and_ "raw". That's a non-ideal implementation. However, to change it -- to require users to now explicitly and separately ingest "raw" -- would be a breaking change to the user-facing API. And perhaps one worth making. But one which would require discussion, design, implmentation, and a deprecation cycle.